protos/processor.proto

```diff
@@ -22,9 +22,11 @@ import "transaction.proto";
 
 
 // The registration request from the transaction processor to the
-// validator/executor
+// validator/executor.
```

Comment:
The commit message is not accurate for this commit. It talks about the MAX_SDK_PROTOCOL_VERSION which is created in the validator. Please remove.

Reply:
Removed this part of the commit message